Main.java

```diff
@@ -9,7 +10,30 @@
    * @param args the command line arguments
    */
   public static void main(String[] args) {
-    // TODO code application logic here
-    
+    // create a scanner for user input
+Scanner input = new Scanner(System.in);
+
+//define if 5 is odd or even
+
+final int odd = 1;
+
+//define if 12632 is odd or even
+final int even = 0;
+
+if(5%2 >0){
+//tell them their number is odd
+ System.out.println("Your number is odd");
+} else {
+//tell them there number is even
+System.out.println("Your number is even");
+}
+if(12632%2 >0){
+//tell them their number is odd
+ System.out.println("Your number is odd");
+} else {
+//tell them there number is even
+System.out.println("Your number is even");
+  }
```

Comment:
Watch your indentation level here. Everything inside the main method should be 2 "tabs" over. Anything inside an if statement should also then have an extra tab. Also, you are missing all of the user input here. The idea is that the user enters in a number and then it will tell them if it is odd or even. You have just hard coded the examples.